CLUSTER.md

```diff
@@ -0,0 +1,31 @@
+# GTSFM on CLUSTER
+
+### How to Run GTSfM on a Cluster?
+
+GTSfM uses [SSHCluster](https://docs.dask.org/en/stable/deploying-ssh.html#dask.distributed.SSHCluster) module of [Dask.distributed](https://distributed.dask.org/en/stable/) library to allow utilizing clusters for GTSfM runs. This is a step-by-step guide on how to set up your machines for a successful run on a cluster.
+
+1. Enable passwordless SSH between all the workers on the cluster
+    - Log in into a machine
+    - For each of the other workers on the cluster run
+        * ssh-copy-id
+        * username@machine_ip_address_of_another_worker
+        * Repeat the above two steps on all machines
+    - Note machines should be able to ssh into themselves passwordless e.g. host1 should be able to ssh into host1.
+2. Clone gtsfm and follow the main readme file to setup the environment on all nodes in the cluster at an identical path
```

Comment:
If we could provide the exact command here, could make this readme even simpler. I think we can assume all machines on the cluster would have to run Linux? does this seem like a fair assumption?

```bash
git clone https://github.com/borglab/gtsfm.git
conda env create -f environment_linux.yml
conda activate gtsfm-v1
```

Reply:
Done.

Reply:
Resolved, thanks!